shared/common-adapters/avatar.render.native.js

```diff
@@ -32,77 +29,63 @@ const sizeToTeamBorderRadius = {
 // Android doesn't handle background colors border radius setting
 const backgroundOffset = 1
 
-class Background extends React.PureComponent<{loaded: boolean, loadingColor: any, borderRadius: number}> {
-  render() {
-    return (
-      <Box
-        loadingColor={this.props.loadingColor}
-        borderRadius={this.props.borderRadius}
-        style={[
-          styles.background,
-          {
-            backgroundColor: this.props.loaded
-              ? globalColors.white
-              : this.props.loadingColor || globalColors.lightGrey,
-            borderRadius: this.props.borderRadius,
-          },
-        ]}
-      />
-    )
-  }
-}
-
-class UserImage extends React.PureComponent<ImageProps> {
-  render() {
-    const {borderRadius, opacity = 1} = this.props
-    return (
-      <NativeImage
-        source={this.props.url}
-        onLoadEnd={this.props.onLoadEnd}
-        style={[styles[`image:${this.props.size}`], {borderRadius, opacity}]}
-      />
-    )
-  }
-}
+const background = props => (
```

Comment:
Seems like `PureComponent` would render less than this, do you want to use `React.memo` for these?